jvm/src/test/kotlin/org/amshove/kluent/tests/assertions/bigdecimal/BigDecimalShouldEqualToShould.kt

```diff
@@ -0,0 +1,25 @@
+package org.amshove.kluent.tests.assertions.bigdecimal
+
+import org.amshove.kluent.shouldEqualTo
+import java.math.BigDecimal
+import kotlin.test.Test
+import kotlin.test.assertFails
+
+
+class BigDecimalShouldEqualToShould {
+
+    @Test
+    fun passWhenComparingTwoEqualValues() {
+        val a = BigDecimal(1925112616126126)
+        val b = BigDecimal(1925112616126126)
+        a.shouldEqualTo(b)
+    }
```

Comment:
Add following test and it will fail in your current implementation: 
```Kotlin
    @Test
    fun passWhenComparingTwoEqualValuesWithDifferentScale() {
        val a = BigDecimal(BigInteger.valueOf(1), -1) // 10
        val b = BigDecimal(BigInteger.valueOf(10)) // 10

        a.shouldEqualTo(b)
    }
```

Reply:
Good catch, that was what I was curious about, thanks :-)